src/demo/ui.jsx

```diff
@@ -203,6 +230,119 @@ class Button extends React.Component {
+        'Garbage dumped in ocean or rivers affects the water health and impacts the marine life in the water.'
+      ]
+    },
+    {
+      heading: 'Train A.I. to Clean Ocean',
+      text: [
+        'In this activity, you will "program" or "train" an artificial intelligence to identify trash to remove from the ocean.'
+      ]
+    },
+    {
+      heading: 'Meet A.I.',
+      text: [
+        "A.I. can't tell if an object is a fish or a piece of trash yet, but it can process different images and identify patterns.",
+        'To program A.I., label the images we show you as either "fish" or "not fish". This will train A.I. to do it on its own!'
+      ]
+    }
+  ],
+
+  short: [
+    {
+      heading: 'Training Data',
+      text: [
+        'A.I. needs lots of training data to do its job well. When you train A.I., the data you provide can make a difference!'
+      ]
+    },
+    {
+      heading: 'Training Data',
+      text: [
+        'We learned how AI and machine learning can be used to do good things like identify trash in the ocean!',
+        'What else can we use AI to do?'
+      ]
+    },
+    {
+      heading: 'Training Data',
+      text: [
+        'AI and machine learning can also be used to give recommendations, like when a computer suggests videos to watch or products to buy.',
+        'Next, you’re going to teach A.I. a new word just by showing examples of that type of fish.'
+      ]
+    }
+  ]
+};
+
+class Instructions extends React.Component {
+  setInstructionsPage(page) {
+    setState({currentInstructionsPage: page});
+  }
+
+  render() {
+    const state = getState();
+    const currentPage = state.currentInstructionsPage;
+    const [, appModeVariant] = getAppMode();
+
+    return (
+      <Body>
+        <Header>{instructionsText[appModeVariant][currentPage].heading}</Header>
+        <div style={styles.instructionsText}>
+          {instructionsText[appModeVariant][currentPage].text.map(
+            (instruction, index) => {
+              return <p key={index}>{instruction}</p>;
+            }
+          )}
+        </div>
+        {instructionsText[appModeVariant].length > 1 && (
+          <div style={styles.instructionsDots}>
+            {instructionsText[appModeVariant].map((instruction, index) => {
+              return (
+                <span
+                  style={
+                    index === currentPage
+                      ? styles.activeDot
+                      : styles.inactiveDot
+                  }
+                  key={index}
+                  onClick={() =>
+                    index !== currentPage
+                      ? this.setInstructionsPage(index)
+                      : null
+                  }
+                >
+                  {'\u25CF'}
+                </span>
+              );
+            })}
+          </div>
+        )}
+        <Footer>
+          <Button style={styles.continueButton} onClick={() => {}}>
```

Comment:
should we just remove the continue button since it's not doing anything right now? (i don't feel very strongly about this)

Reply:
Good point.  I think it's most likely we'll have a button here, so I thought it worth leaving in so that we have an idea how all the UI elements will play together.